src/test/java/org/dependencytrack/policy/cel/CelPolicyEngineTest.java

```diff
@@ -295,7 +295,6 @@ public void testEvaluateProjectWithAllFields() {
                   && project.name == "projectName"
                   && project.version == "projectVersion"
                   && project.classifier == "APPLICATION"
-                  && project.is_active
```

Comment:
Can we keep the `is_active` check here please? Since the test's purpose is to verify that *all* fields are loaded correctly. I think it would need to be changed to `!project.is_active` now, since `inactiveSince` is being populated above.